web-server/v0.4/src/pages/ComparisonSelect/index.js

```diff
@@ -133,7 +135,15 @@ class ComparisonSelect extends React.Component {
               };
               return (
                 <div key={iteration.resultName} style={{ marginTop: 32 }}>
-                  <h2>{iteration.resultName}</h2>
+                  <h2>
+                    {iteration.resultName}{' '}
```

Comment:
We should use CSS for all styling/formatting as opposed to inserting whitespace into the JSX. I highly recommend CSS Flexbox for the kind of structured spacing we are looking for here: https://www.w3schools.com/css/css3_flexbox.asp